src/jenkins.py

```diff
@@ -54,6 +54,8 @@
 
 USER = "jenkins"
 GROUP = "jenkins"
+UID = "2000"
```

Comment:
These values are only used in tests. Can you define them in the test sources instead?

Reply:
Thanks, I moved them to the test file